local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -739,8 +767,11 @@ void LocalPlannerNode::printPointInfo(double x, double y, double z) {
   printf("-------------------------------------------- \n");
 }
 
-void LocalPlannerNode::pointCloudCallback(const sensor_msgs::PointCloud2& msg) {
-  newest_point_cloud_ = msg; // FIXME: avoid a copy
+void LocalPlannerNode::pointCloudCallback(const sensor_msgs::PointCloud2::ConstPtr& msg, int index) {
+
+  available_clouds_[index] = true;
```

Comment:
I think this booleans are never reset to false

Reply:
but with the arbitrary camera number this variable has been eliminated